framework/src/main/java/org/apache/drill/test/framework/TestDriver.java

```diff
@@ -65,7 +67,9 @@
   private Connection connection = null;
   private ConnectionPool connectionPool = null;
   private String commitId;
-  private String version;
+  private static String version;
+  private String[] injectionKeys = {"DRILL_VERSION"};
```

Comment:
This is OK. However I think an enum is more appropriate for injectionKeys as we can avoid using the hardcoded constants ("DRILL_VERSION") directly in other places of the code